src/uu/kill/src/kill.rs

```diff
@@ -69,10 +71,10 @@ pub fn uumain(args: impl uucore::Args) -> i32 {
             return kill(&sig, &pids_or_signals);
         }
         Mode::Table => table(),
-        Mode::List => list(pids_or_signals.get(0).cloned()),
+        Mode::List => list(pids_or_signals.get(0).cloned())?,
```

Comment:
I think this `match` could just be the last expression in the function:
```rust
match mode {
    Mode::Kill = > {
        ...
        kill(&sig, &pids_or_signals)
    }
    Mode::Table => table(),
    Mode::List => list(pids_or_signals.get(0).cloned()),
}

Reply:
Added `UResult` for `table` to make this work.